### PR TITLE
Basics of Transfer Learning

### PR DESCRIPTION
**Pull request by collins-a**
-Article on transfer learning as index.md
-Hero image as hero.jpg
-Image of transfer learning strategies as strategies.jpg

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      

#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tool:** [Grammarly](https://grammarly.com)
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by collins-a** [comment]
This fixes #451

**Comment by hectorkambow** [comment]
Plagiarism check done ✔️

File: articles/basics-of-transfer-learning/index.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+Relational-knowledge transfer maps out relational knowledge between source and target domains. This method involves data that is not independent and identically distributed. For example, networked data and social network data. The technique looks to transfer the relationship within data from a source to a target domain.
+
+For a deep dive into these transfer learning approaches, take a look at this [paper]( https://www.cse.ust.hk/~qyang/Docs/2009/tkde_transfer_learning.pdf).
+
+### Applications of Transfer Learning
+Several transfer learning algorithms are used in other related fields. Such as medicine, image detection, speech recognition, and recommendation systems to name a few. We will go over these in more detail in the following sections.
+
+#### Medicine
+Many algorithms have been proposed and implemented in medical imaging to reduce the workload of doctors. They also serve the purpose of improving the accuracy of judgment. One method involves using a [convolutional neural network (CNN)](https://en.wikipedia.org/wiki/Convolutional_neural_network) for model transfer. The goal is the segmentation of white matter lesions of multiple sclerosis (MS) in patient images.
```
Comment (lalith1403) [inline articles/basics-of-transfer-learning/index.md:124]: The problem statement needs some attention. Consider covid-19 detection using X-rays as an example or any other example that you find suitable. Restructure the problem statement to enhance readability.

File: articles/basics-of-transfer-learning/index.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+An approach involves segmentation tasks of chest X-ray images using domain adaptation. Domain adaptation refers to where a model is trained on a different source domain from the target domain. Both domains are related.
+
+Another method involves the use of adversarial domain adaptation. This method is used to classify prostate histopathology whole-slide images. Adversarial domain adaptation is a method used to transform the features of many target domains to be uniform with source domain features.
```
Comment (lalith1403) [inline articles/basics-of-transfer-learning/index.md:128]: I feel the examples are too specific. You should take the approach of mentioning the ML algorithm and a use-case for the algorithm. As a techie will be reading this article, they will drift away from the article unless they are into medicine.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
@collins-a Amazing article. A few minor suggestions to enhance readability are given. I have also mentioned a few doubts that I got as a reader. Do let me know if you feel some of the questions are answered in the article. 

After making the requested changes, a Grammarly check would ensure minimal errors. 

I loved the part on types of transfer learning. Got me thinking, which is what is required from an article. The article should be thought-provoking and at the same time, giving the answers. Excited to read the updated article.

File: articles/basics-of-transfer-learning/index.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+### Useful Terms
+**Pre-trained models** – models trained on a sizeable benchmark dataset to solve a problem similar to a problem one might want to solve.
+
+**Neural network** – a series of algorithms that are modeled on the human brain, to identify underlying relationships in data.
+
+**Artificial general intelligence (AGI)** – hypothetical machine intelligence with the capacity to learn or understand any intellectual task a human being can.
+
+**Domain knowledge** – knowledge about the environment in which a target system operates.
+
+**Feature** – an individual measurable property that is being observed. It is an attribute shared by all the independent units on that analysis is to be done.
```
[the remark this answers is not in the document]
Comment (hectorkambow) [reply articles/basics-of-transfer-learning/index.md:42]: I may have changed this - I change most which to that's  (is just more common) 
👍

Comment (lalith1403) [reply articles/basics-of-transfer-learning/index.md:42]: Sorry about that. I have deleted the comment. Thank you.

**Comment by hectorkambow** [comment]
@collins-a  just pining you to make sure you saw the comments by our peer reviewer 🚀

**Comment by collins-a** [comment]
Hey, @hectorkambow @lalith1403! Apologies for the delay, made a number of improvements to the article. Let me know if they are sufficient.

**Review by lalith1403 — COMMENTED** [review]
Great job @collins-a

**Pull request merged**